packages/account/src/Components/personal-details/__tests__/personal-details.spec.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
-import { fireEvent, render, screen, waitFor, within } from '@testing-library/react';
+import { act, fireEvent, render, screen, waitFor, within } from '@testing-library/react';
```

Comment:
I can’t find the usage of 'act' in test cases, could ypu please clarify on this?

Reply:
Was testing state update and missed it there. Will remove. Thanks